README.md

```diff
@@ -25,22 +24,20 @@ We recommend you install [pre-commit](https://pre-commit.com/) and install the p
 pre-commit install
 ```
 
-If you add new dependencies to the project, put them in the `setup.cfg` file and regenerate the `requirements.txt`
+If you add new dependencies to the project, put them in the `pyproject.toml` file and regenerate the `requirements.txt`
```

Comment:
Was deleting all of the existing requirements files intentional?

Reply:
Same thought -- I follow the removal of requirement lockfiles except for removing the lockfile for `-test.txt`?

Reply:
Yes. It's a library - libraries should specify ranges of dependencies they support, not locked to a particular version. It's up to the services using the library to lock to a particular version - otherwise you will very quickly run into dependency conflicts.

Reply:
Got it-- thanks for the explanation.

Reply:
This doesn't explain the removal of the requirements-test file? We operate in a non-library mode when the test suite is run.

Reply:
Conversely, it feels unwise to me to roll without a lockfile in this situation. If we leave unpinned we could have a test suite that passes when it's installed and run on day 0 and the same suite doesn't pass when it's installed and run on day 40. A contributor could add a feature, run the suite, and it fails but they think that's due to their addition (kinda similar to what happened to me with the unpinned `cosign` -- an incompatible major version released between when the last time he Dockerfile was built and when I tried to build yesterday).

I think the artifact of lockfile at the point in time a suite passed is important.

A compromise that could address both concerns seems like it could be to re-build periodically, generating that lockfile anew each month or so and committing that.

Note that this isn't blocking for me -- this PR gets a lot of very handy value adds in that I do think are worth it.